### DISCUSSION TITLE
Add cosmetic settings for PerspectiveWidget

### DISCUSSION THREAD

**Discussion by wangkev** [perspective-dev/perspective#1864] [Q&A] [answered]
### Description of Problem:
Would like to specify cosmetic settings for PerspectiveWidget. For example: control which y-axis to use, float formatting, collapsed pivot level.

**Comment by timkpaine** [comment, marked as answer]
This is doable via the `plugin_config` attribute:

Y Axis:
`{'splitMainValues': ['Category']}`


Float config:
`{'columns': {'YOUR COLUMN': {'fixed': NUMBER OF DECIMAL PLACES}}}`

**Comment by texodus** [comment]
You can also inspect the config object of any `<perspective-viewer>` by choosing the `config` option in the Export or Copy menus:

<img width="226" alt="Screen Shot 2022-07-13 at 2 24 03 PM" src="https://user-images.githubusercontent.com/60666/178804469-d183b5fe-dec9-42e8-8cc2-a06c33f75ecf.png">

**Reply by timkpaine** [reply]
we might consider adding the symmetric "upload settings from config json" as a context-agnostic save/restore function (when you haven't implemented that into your own app)
